bbreaker/GameObject.py

```diff
@@ -18,46 +19,71 @@ def __init__(self, bricks_x=7, bricks_y=11, brick_size=60):
 
         self.settings = {'bricks_number': (bricks_x, bricks_y),
                          'brick_size': brick_size,
-                         'screen_size': (brick_size*bricks_x,
-                                         brick_size*bricks_y,)}
+                         'screen_size': (brick_size * bricks_x,
+                                         brick_size * bricks_y,)}
 
         self.surface = pygame.display.set_mode(self.settings['screen_size'])
         self.game_on = False
         self.next_level = True
         self.level = 0
         Ball.shot_timer = pygame.USEREVENT + 1
         Ball.shot_counter = 0
-        Ball.init_position = [self.settings['screen_size'][0]//2,
-                              (self.settings['bricks_number'][1] - 1)
-                              * self.settings['brick_size']]
+        Ball.init_position = [self.settings['screen_size'][0] // 2,
+                              (self.settings['bricks_number'][1] - 1) * self.settings['brick_size']]
+
+        # set cursor type
+        pygame.mouse.set_cursor(*pygame.cursors.broken_x)
+        self.current_mpos = Ball.init_position
 
     def update(self):
         color = (20, 10, 30)
         self.surface.fill(color)
         self.balls.update(self.surface)
         self.bricks.update(self.surface)
-        pygame.display.update()
+
         if {ball.game_on for ball in self.balls} == {False}:
             self.game_on = False
 
+        # draw aim line
```

Comment:
It would be nice to have a separate function for drawing a line and then sth like: `self.shot_line.update(self.surface)`.